apps/system/js/notifications.js

```diff
@@ -460,6 +464,17 @@ var NotificationScreen = {
     messageContent.setAttribute('dir', 'auto');
     message.appendChild(messageContent);
     notificationNode.appendChild(message);
+    var countDiv = document.createElement('div');
+    countDiv.classList.add('detail-count');
+    countDiv.textContent = "1";
+    notificationNode.appendChild(countDiv);
+  } else {
+    var notificationNode = document.querySelector('.notification');
+    var count = document.getElementById(detail.appName).querySelector('.detail-count').textContent;
```

Comment:
This is a terrible idea.  We can't use the HTML view to store data that belongs in the model. You can presumably figure out the correct count of notifications by searching the unreadNotifications array or something.  Please change the way you store the notification data so that you can figure out the count without refering to the HTML.